### PR TITLE
add `ceremonies_getReputation` rpc

### PR DESCRIPTION
**Pull request by clangenb**

### CODE REVIEW

**Comment by clangenb** [comment]
@NicJak when you are ready, can you do the same thing for the following rpc PRs of yours?

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -595,6 +596,13 @@ impl_runtime_apis! {
 		}
 	}
 
+
+	impl pallet_encointer_ceremonies_rpc_runtime_api::CeremoniesApi<Block, AccountId> for Runtime {
+		fn get_reputations() -> Vec<(CommunityCeremony, AccountId, Reputation)> {
+			EncointerCeremonies::get_reputations()
```
Comment (brenzi) [inline runtime/src/lib.rs:602]: Why plural? Does this return reputation for all accounts? Why?

Comment (brenzi) [reply runtime/src/lib.rs:602]: after discussion with @clangenb I believe we should merge this as is and accept bad performance for now. The solution will be slightly more involved than I thought:
https://github.com/encointer/pallets/issues/118

**Review by brenzi — APPROVED** [review]

**Comment by brenzi** [comment]
I doubt this has ever worked....how did you test it?
this should return reputation for a specified account, while the pallet only allows fetching reputation for all accounts. there is a mismatch currently, I believe

**Pull request merged**